src/profile-logic/marker-schema.js

```diff
@@ -537,17 +543,20 @@ export function formatMarkupFromMarkerSchema(
         </ul>
       );
     case 'url':
+      if (!value.startsWith('http:') && !value.startsWith('https:')) {
+        return value;
+      }
```

Comment:
Why? So `localhost:4242/index` would not be clickable?

Reply:
I would like to remove the `http[s]://` part, because it does not offer any information.

Reply:
Good idea, I'll look at it tomorrow